### DISCUSSION TITLE
Bucket4j JDBC Integration with Spring Test

### DISCUSSION THREAD

**Discussion by BarracudaX** [vladimir-bukhtoyarov/bucket4j#440] [Q&A]
When using bucket4j jdbc integration, is there a way to make bucket4j jdbc integration participate in existing transaction like the one created when using Spring Test.
I tried running spring test with bucket4j, but it seems that bucket4j creates and commits transaction internally.

**Comment by vladimir-bukhtoyarov** [comment]
No, it is not possible.
